go/arrow/table.go

```diff
@@ -42,20 +43,19 @@ type Table interface {
 // To get strongly typed data from a Column, you need to iterate the
 // chunks and type assert each individual Array. For example:
 //
-// 		switch column.DataType().ID {
-//		case arrow.INT32:
-//			for _, c := range column.Data().Chunks() {
-//				arr := c.(*array.Int32)
-//				// do something with arr
-//			}
-//		case arrow.INT64:
-//			for _, c := range column.Data().Chunks() {
-//				arr := c.(*array.Int64)
-//				// do something with arr
-//			}
-//		case ...
+//	switch column.DataType().ID {
```

Comment:
This block was formatted 'badly' according to `goimports`?

Reply:
interesting. *shrug* nothing wrong with this modification honestly.